src/app_service/service/community_tokens/service.nim

```diff
@@ -138,3 +143,19 @@ QtObject:
       if token.symbol == symbol:
         return token
 
+  proc contractOwner*(self: Service, chainId: int, contractAddress: string): string =
+    try:
+      let response = tokens_backend.contractOwner(chainId, contractAddress)
+      return response.result.getStr()
+    except RpcException:
+      error "Error getting contract owner", message = getCurrentExceptionMsg()
+
+  proc airdropCollectibles*(self: Service, communityId: string, password: string, collectiblesAndAmounts: seq[CommunityTokenAndAmount], walletAddresses: seq[string]) =
+    try:
+      for collectibleAndAmount in collectiblesAndAmounts:
+        let addressFrom = self.contractOwner(collectibleAndAmount.communityToken.chainId, collectibleAndAmount.communityToken.address)
+        let txData = TransactionDataDto(source: parseAddress(addressFrom)) #TODO estimate fee in UI
+        let response = tokens_backend.mintTo(collectibleAndAmount.communityToken.chainId, collectibleAndAmount.communityToken.address, %txData, password, walletAddresses, collectibleAndAmount.amount)
+        echo "!!! Transaction hash ", response.result.getStr()
```

Comment:
Leftover. Should we check for errors or send an event that the TX was sent?

Reply:
In case of error exception will be thrown. I keep this echo for debugging purposes. I will remove it later.
We will also add notifications later. The UI is minimal and we do not show status of airdrop.